plugins/module_utils/network/ios/config/l3_interfaces/l3_interfaces.py

```diff
@@ -141,7 +141,7 @@ def _state_replaced(self, want, have, module):
 
         for interface in want:
             for each in have:
-                if each["name"] == interface["name"]:
+                if each["name"].lower() == interface["name"].lower():
```

Comment:
Wouldn't this get solved by passing the want through normalize_interface() before calling set_state()? Seems more robust than having to deal with this everywhere separately.

Reply:
yea, make sense